lib/chef/mixin/homebrew_user.rb

```diff
@@ -60,15 +60,15 @@ def find_homebrew_username(provided_user = nil)
       private
 
       def calculate_owner
-        default_brew_path = "/usr/local/bin/brew"
+        default_brew_path = RbConfig::CONFIG["host_cpu"].eql?("arm64") ? "/opt/homebrew/bin/brew" : "/usr/local/bin/brew"
```

Comment:
why not just use our internal `which` library (https://github.com/chef/chef/blob/main/lib/chef/mixin/which.rb) rather than any of this?

Reply:
For future-proofing, I would do:

```ruby
[which('brew'), '/opt/homebrew/bin/brew', '/usr/local/bin/brew].select { |x| File.exist?(x) }.first
```

Reply:
fixed in latest commit